tests/dataframe_test.py

```diff
@@ -347,6 +346,15 @@ def test_head(self):
         hdf_ref = ref_df.head(3).reset_index(drop=True)
         self.assertTrue(hdf_ref.equals(hdf.to_pandas()))
 
+    def test_first(self):
+        "special case of the head function, only calling the first row of the data frame"
+        df = build_ak_df()
+        ref_df = build_pd_df()
+
+        firstdf = df.first
+        firstdf_ref = ref_df.head(1).reset_index(drop=True)
```

Comment:
@pierce314159 - `groupby.first` would be the preference. I believe there were a few other functions as well. However, since the users seems satisfied with DataFrame, I am not worried about adding in the other functionalities right now.